### DISCUSSION TITLE
How does one generate the avro

### DISCUSSION THREAD

**Discussion by gnatcode** [hamba/avro#399] [Q&A]
I see the avrogen creates an object model from a schema for GO but is there away to do the reverse and create the avro schema from an object using avrogen? If so Its not clear in the documentation any advise would be appreciated.

**Comment by nrwiersma** [comment]
No, and this is not a planned feature. You can check out https://github.com/hamba/avro/issues/263 for the discussion around this, perhaps it puts you on the correct path.
